### ISSUE TITLE
module QuPath

### ISSUE THREAD

**Issue by heylf** [heylf/spatialxe#9]
### Description of feature

# Description
https://qupath.github.io/
https://www.10xgenomics.com/support/software/xenium-explorer/latest/tutorials/xe-image-file-conversion

Can we implement this?

**Comment by jwrth** [comment]
Hi @heylf, what exactly was your idea to implement here? A general conversion of image files from different formats to Xenium Explorer compatible formats? I already implemented the function that is shown on the 10x website in my analysis workflow and use it regularly.

**Comment by heylf** [comment]
Yes exactly

**Comment by jwrth** [comment]
Yes I think that might be useful and relatively easy to achieve with a combination of [`aicsimageio`](https://github.com/AllenCellModeling/aicsimageio) and [`slideio`](https://pypi.org/project/slideio/). 

However, it might be even more interesting to implement something that also registers such post-Xenium stained images to the Xenium data. I already have a `opencv`-based workflow set up to automatically register images to xenium data. I think it might be very useful to the community to implement this as a nf-core module.

My workflow currently is structured like that:
1. Selection and naming of individual regions on post-Xenium stained image in QuPath:
![grafik](https://github.com/heylf/spatialxe/assets/76480183/54f15c70-78c7-4afc-b493-c9780c74f441)
2. Export of the regions using a script in QuPath
3. Registration of images to xenium data using something like `register_images(data=xenium_data, image_dir=path/to/images)

By using a fixed naming scheme in the export, I think we could implement this in a quite generalized way which would work then also if there are multiple post-Xenium stainings of the same Xenium slide (e.g. HE, IHC and IF). And I think this would be very useful because the manual alignment in Xenium explorer can be very painful...
What do you think?

**Comment by heylf** [comment]
That would be awesome. We could make then two distinct tracks in the pipeline. One for preprocessing and one for re-processing of the data.
